lib/ovirt/vm.rb

```diff
@@ -80,6 +80,18 @@ def move(storage_domain)
       action['href']
     end
 
+    def boot_order=(devices)
+      devices = [ devices ].flatten
+      update! do |xml|
+        xml.os do
+          devices.each do |d|
+            xml.boot(:dev => d)
+          end
+        end
+        xml.memory self[:memory]  # HACK: RHEVM BUG: RHEVM resets it to 10GB, unless we set it
```

Comment:
Let's remove it.  If we need it, we can add it later (with a better description of the problem).